scripts/build.ps1

```diff
@@ -402,7 +414,7 @@ function Create-NugetPackages
         }
 
         Write-Verbose "$nugetExe pack $stagingDir\$file -OutputDirectory $packageOutputDir -Version $TPB_Version -Properties Version=$TPB_Version $additionalArgs"
-        & $nugetExe pack $stagingDir\$file -OutputDirectory $packageOutputDir -Version $TPB_Version -Properties Version=$TPB_Version`;Runtime=$TPB_TargetRuntime`;NetCoreTargetFramework=$TPB_TargetFrameworkCore20 $additionalArgs
+        & $nugetExe pack $stagingDir\$file -OutputDirectory $packageOutputDir -Version $TPB_Version -Properties Version=$TPB_Version`;Runtime=$TPB_TargetRuntime`;NetCoreTargetFramework=$TPB_TargetFrameworkCore20`;UapTargetFramework=$TPB_TargetFrameworkNS1_4 $additionalArgs
```

Comment:
Where are we using "UapTargetFramework"?

Reply:
In ObjectModel nuspec

Reply:
This may need CI.Real definition change.

Reply:
I'm not sure why, since this variable is created via build.ps1, which then uses it via invoking nuget.exe to create packages, so I think it would work fine